### DISCUSSION TITLE
Add conditions to property editors

### DISCUSSION THREAD

**Discussion by ccasalicchio** [umbraco/Umbraco-CMS#14212] [Features and ideas]
It would be really nice to be able to set conditions for property editors, based on the selection of other property editors in the same content.
For example:
A property that allows editors to add a label to an item
![image](https://user-images.githubusercontent.com/2466879/236510420-c84be810-7693-4dfc-ae8a-47182c1d658a.png)

from that selection, a textbox appears (or disappears) allowing the editor to add the label text
![image](https://user-images.githubusercontent.com/2466879/236510766-5ccc89c1-d851-4564-b370-d5fab2d8a2a8.png)

So, in the content editor, by default the label textbox is hidden, and only appears if the "Has Label" toggle has been set to "ON"
![image](https://user-images.githubusercontent.com/2466879/236513527-09f815ce-63cd-4ef5-b6b2-a8243dbe7fc4.png)
![image](https://user-images.githubusercontent.com/2466879/236513605-fdee64ad-1061-4ebd-bcaf-de58cf8fe480.png)

**Comment by skttl** [comment]
Theres a package that does this, if you need it now. https://marketplace.umbraco.com/package/our.umbraco.conditionaldisplayers

**Reply by ccasalicchio** [reply]
That's great! thank you for the info!
